### PR TITLE
Fixed-DatabaseConnection

### PR DESCRIPTION
**Pull request by Karthi-DStech**
Hi James, 

I've removed the executing logic of closing the database connection from OMOP_match.py to text_input.py. 

This is because, Lettuce treats the words individually and yields the results. I've tried mulitple times and techniques to solve the issue. 

Then I though of implementig this solution and worked well (as it was pretty straight forward and gives a solution). 

I've also tested the code and it works as expected. Kindly review the code and do the neetful. 

Thanks and regards,
Karthik

### CODE REVIEW

**Review by kuraisle — CHANGES_REQUESTED** [review]
This is an interesting solution. The issue I have with it is that the OMOPMatcher singleton is called from the UI, so is created client-side. This restricts deployment: if we run a LLettuce server, and have people open up the UI and call our API, this function won't work unless they have the authentication details for the OMOP server, which is not a behaviour we want

**Comment by Karthi-DStech** [comment]
> This is an interesting solution. The issue I have with it is that the OMOPMatcher singleton is called from the UI, so is created client-side. This restricts deployment: if we run a LLettuce server, and have people open up the UI and call our API, this function won't work unless they have the authentication details for the OMOP server, which is not a behaviour we want

Thanks for your feedback. Actually, I tried not to modify the base logic as much as I could and If we want to implement the logic in OMOP matcher and app.py, we have to remove the yield and modify some logic. I'll start working on it and let you know the progress.

**Comment by kuraisle** [comment]
Can't you just pass a list to OMOP_match.run? OMOPMatcher.calculate_best_matches takes a list

**Comment by Karthi-DStech** [comment]
> Can't you just pass a list to OMOP_match.run? OMOPMatcher.calculate_best_matches takes a list

I've done the implementation in another pull request James. This was before fixing the multiple query commit.

**Comment by Karthi-DStech** [comment]
Hi James, 

I've made changes to the database connection and it works well. I've moved the logic to `app.py` and tested the code. Kindly review it and do the needful. 

Thanks and regards,
Karthik

File: Carrot-Assistant/app.py [hunk 1]
```diff
@@ -39,16 +40,17 @@
...
     """
     This enum holds the names of the different models the assistant can use
     """
 
-    GPT_3_5_TURBO = "gpt-3.5-turbo-0125",
-    GPT_4 = "gpt-4",
-    LLAMA_2_7B = "llama-2-7B-chat",
-    LLAMA_3_8B = "llama-3-8B",
-    LLAMA_3_70B = "llama-3-70B",
+    GPT_3_5_TURBO = ("gpt-3.5-turbo-0125",)
```
Comment (kuraisle) [inline Carrot-Assistant/app.py:49]: These shouldn't be wrapped in tuples

**Pull request closed without merging**